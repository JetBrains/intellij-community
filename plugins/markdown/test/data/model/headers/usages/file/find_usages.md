# Header<caret> name

[Link to header](#header-name)

Some list:
* First list item
* Item with link [link to header](#header-name)

<!-- TOC -->
* [Some header](#header-name)
<!-- TOC -->


|     | this is header                         |     |
|-----|----------------------------------------|-----|
|     | [some link inside table](header-name) |     |
|     |                                        |     |

[Other link to header](#header-name)
[link with uppercase](#header-NAME)
