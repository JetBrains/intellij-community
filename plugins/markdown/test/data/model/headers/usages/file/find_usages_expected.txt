10|* |[Some header](#|header-name|)
19|[Other link to header](#|header-name|)
20|[link with uppercase](#|header-NAME|)
3|[Link to header](#|header-name|)
7|* |Item with link [link to header](#|header-name|)
